packages/next/src/utils.ts

```diff
@@ -2202,6 +2202,7 @@ interface BaseEdgeFunctionInfo {
   page: string;
   wasm?: { filePath: string; name: string }[];
   assets?: { filePath: string; name: string }[];
+  userConfig?: Record<string, unknown>;
```

Comment:
```suggestion
  userConfig?: {
    regions?: 'auto' | string[] | 'all' | 'default';
    [key: string]: unknown;
  };
```

This will make TS happy